<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerticalActuatorWidget</class>
 <widget class="QWidget" name="VerticalActuatorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VerticalActuatorWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="foo_0">
     <item>
      <widget class="QLabel" name="foo_device_label">
       <property name="text">
        <string>Axis</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="AxisComboBox">
       <item>
        <property name="text">
         <string>Axis 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Axis 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Horizontal Axis</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="foo_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="TargetSpinBox">
       <property name="maximum">
        <double>110000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="foo_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Threshold</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ThresholdSpinBox">
       <property name="maximum">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
